include/llvm/Support/Program.h

```diff
@@ -33,7 +34,7 @@ namespace sys {
 /// @brief This struct encapsulates information about a process.
 struct ProcessInfo {
 #if defined(LLVM_ON_UNIX)
-  typedef pid_t ProcessId;
+  typedef llbuild_pid_t ProcessId;
 #elif defined(LLVM_ON_WIN32)
   typedef unsigned long ProcessId; // Must match the type of DWORD on Windows.
```

Comment:
Actually, noticing this now (sorry), why not just re-use the existing ProcessId typedef from here?

Reply:
It is nested in a struct, so it's not visible from Swift :/